site/content/en/blog/releases/1.17.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.17.0 - Kubernetes 1.20, deprecated required/preferred, add sync property to FleetAutoScaler"
+linkTitle: "1.17.0"
+date: "2021-08-31"
+---
+
+
+This is the 1.17.0 release of Agones.
+
+This release brings us support for Kubernetes 1.20, so if you are testing this release candidate, remember to upgrade your clusters.
```

Comment:
nit: Should be "installing this release" instead of "testing this release candidate" since this is the final release.